gnome_search_provider/src/main.rs

```diff
@@ -0,0 +1,150 @@
+use arboard::Clipboard;
+use ripasso::pass::PasswordStore;
+use search_provider::{ResultID, ResultMeta, SearchProvider, SearchProviderImpl};
+use std::{collections::HashMap, path::PathBuf, thread, time};
+use zbus::{blocking::Connection, proxy, zvariant::Value};
+use zeroize::Zeroize;
+
+#[proxy(
+    default_service = "org.freedesktop.Notifications",
+    default_path = "/org/freedesktop/Notifications"
+)]
+trait Notifications {
+    fn notify(
```

Comment:
Annotate it with #[allow(clippy::too_many_arguments)] in order to make clippy happy